src/op_analytics/coreutils/request.py

```diff
@@ -55,12 +56,12 @@ def get_data(
         with attempt:
             if attempt.num > 1:
                 log.warning(f"retrying {url}")
-            return _get_data(session, url, headers)
+            return _get_data(session, url, headers, params)
 
 
-def _get_data(session: requests.Session, url: str, headers: dict[str, str]):
+def _get_data(session: requests.Session, url: str, headers: dict[str, str], params: dict[str, str]):
```

Comment:
I think the type hint for params should be `dict[str, Any]`. In case param values are not strings (for example ints or lists).